massa-execution-worker/Cargo.toml

```diff
@@ -21,7 +21,7 @@ massa_execution_exports = { path = "../massa-execution-exports" }
 massa_models = { path = "../massa-models" }
 massa_storage = { path = "../massa-storage" }
 massa_hash = { path = "../massa-hash" }
-massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", tag = "v0.8.2" }
+massa-sc-runtime = { path = "../../massa-sc-runtime" }
```

Comment:
Just a reminder here to switch to a released version of massa-sc-runtime

Reply:
Yep, just waiting for every corresponding PR to be ready